contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -3655,3 +3655,81 @@ END
 $$;
 GRANT EXECUTE ON PROCEDURE sys.sp_changedbowner(IN sys.sysname, IN sys.VARCHAR(5)) TO PUBLIC;
 
+CREATE OR REPLACE PROCEDURE sys.sp_procedure_params_100_managed(IN "@procedure_name" sys.sysname, 
+                                                                IN "@group_number" integer DEFAULT 1, 
+                                                                IN "@procedure_schema" sys.sysname DEFAULT 'dbo'::sys."varchar", 
+                                                                IN "@parameter_name" sys.sysname DEFAULT NULL::sys."varchar")
+AS $$
+
+BEGIN
+        SELECT 	CAST (v.column_name AS sys.sysname) AS [PARAMETER_NAME],
+				CAST (CASE
+						WHEN v.column_type = 5 THEN 4
+                        WHEN v.column_type = 3 THEN 4
+                        ELSE v.column_type END
+                     AS smallint) AS [PARAMETER_TYPE],
+        		CAST (CASE   
```

Comment:
Instead of using the following T-SQL syntax of CASE:
```
-- Searched CASE expression:
CASE
     WHEN Boolean_expression THEN result_expression [ ...n ]
     [ ELSE else_result_expression ]
END
```
Let's use this one:
```
-- Simple CASE expression:
CASE input_expression
     WHEN when_expression THEN result_expression [ ...n ]
     [ ELSE else_result_expression ]
END
```